service/src/main/java/uk/gov/hmcts/reform/fpl/model/ChildParty.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import uk.gov.hmcts.reform.fpl.model.common.EmailAddress;
+import uk.gov.hmcts.reform.fpl.model.common.Party;
+import uk.gov.hmcts.reform.fpl.model.common.TelephoneNumber;
+
+import java.util.Date;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public final class ChildParty extends Party {
+    private final String gender;
+    private final String genderIdentification;
+    private final String livingSituation;
+    private final String situationDetails;
+    private final Date situationDate;
+    private final String keyDates;
+    private final String careAndContact;
+    private final String adoption;
+    private final String placementOrderApplication;
+    private final String placementCourt;
+    private final String mothersName;
+    private final String fathersName;
+    private final String fathersResponsibility;
+    private final String socialWorkerName;
+    private final TelephoneNumber socialWorkerTel;
+    private final String additionalNeeds;
+    private final String additionalNeedsDetails;
+    private final String detailsHidden;
+    private final String detailsHiddenReason;
+    private final String litigationIssues;
+    private final String litigationIssuesDetails;
+
+    @JsonCreator
```

Comment:
remove all of these @Json annotations. Add @Autowired above where you instantiate ObjectMapper